EDDI/MainWindow.xaml.cs

```diff
@@ -329,7 +331,7 @@ private void windowLoaded(object sender, RoutedEventArgs e)
             {
                 if (Properties.Settings.Default.Minimized)
                     senderWindow.WindowState = WindowState.Minimized;
```

Comment:
Please add braces around all such one-liners. I know it takes up more space but it's a real risk as proven by "goto fail".

Visual Studio can help catch these via its lightbulb tip interface.